scripts/docker/ubuntu.14.04/Dockerfile

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM ubuntu:14.04
+
+# Misc Dependencies for build
+RUN apt-get update && apt-get -qqy install curl unzip gettext sudo
+
+# This could become a "microsoft/coreclr" image, since it just installs the dependencies for CoreCLR (and stdlib)
+RUN echo "deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.6 main"  | tee /etc/apt/sources.list.d/llvm.list && \
+    curl http://llvm.org/apt/llvm-snapshot.gpg.key | apt-key add - && \
+    apt-get update && apt-get -qqy install\
+	libc6 \
```

Comment:
The question really is:  shouldn't the Ubuntu14 and Ubuntu16 docker files look very, very similar?  In fact, I think these two files should share implementation where it makes sense, instead of being:

1. inconsistent
2. duplicated